src/main/resources/swagger/api-docs.yaml

```diff
@@ -3559,6 +3583,7 @@ definitions:
     description: ''
     required:
       - acl
+    type: object
```

Comment:
these changes were made to make our yaml readable by the bravado testing framework